src/ds3_requests.c

```diff
@@ -364,7 +364,7 @@ static ds3_error* _get_request_xml_nodes(
         char* root_element_name) {
     xmlNodePtr root;
 
-    xmlDocPtr doc = xmlParseMemory((const char*) xml_blob->data, xml_blob->len);
+    xmlDocPtr doc = xmlReadMemory((const char*) xml_blob->data, xml_blob->len, NULL, NULL, XML_PARSE_HUGE);
```

Comment:
fyi this is an auto-gened file

Reply:
Ok, we'll need that change reflected there as well.